src/usr/share/gpgmailer/gpgmailer.py

```diff
@@ -14,19 +14,24 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 __all__ = ['GpgMailer']
-__author__ = 'Joel Luellwitz, Emily Frost, and Brittney Scaccia'
+__author__ = 'Joel Luellwitz and Emily Frost'
 __version__ = '0.8'
 
 import base64
 import json
 import logging
 import os
+import subprocess
+import sys
 import time
 import traceback
+from parkbenchcommon import broadcastconsumer
 import gpgkeyverifier
 import gpgmailbuilder
-import mailsender
 
+# The number of seconds to wait after a broadcast to get another broadcast.
+NETCHECK_BROADCAST_DELAY = 5
+NETCHECK_BROADCAST_NAME = 'network-reconnected'
```

Comment:
I probably suggested this name, but our understanding of this has changed. How about "gateway-changed"? Also, I'd name the constants BROADCAST_NETCHECK_GATEWAY_CHANGED_NAME and BROADCAST_NETCHECK_GATEWAY_CHANGED_DELAY.